src/hydra-queue-runner/hydra-queue-runner.cc

```diff
@@ -854,6 +858,25 @@ int main(int argc, char * * argv)
     return handleExceptions(argv[0], [&]() {
         initNix();
 
+        /* Export a simple "up" metric, to allow monitoring that we're
+           still alive. */
+        std::thread([&]() {
+            prometheus::Exposer exposer{"127.0.0.1:8080"};
+
+            // @note it's the users responsibility to keep the object alive
+            auto registry = std::make_shared<prometheus::Registry>();
+
+            auto& running = prometheus::BuildGauge()
+                .Name("hydra_queue_runner_running")
+                .Help("Whether the queue runner is currently running")
+                .Register(*registry);
+
+            exposer.RegisterCollectable(registry);
+            running.Add({}).Set(1);
```

Comment:
Prometheus automatically creates an `up` metric for every scrape target, so just by virtue of having an exposer at all will create this `up` :).